torch/jit/_builtins.py

```diff
@@ -83,8 +83,25 @@
     (torch.nn.init._no_grad_zero_, "aten::_no_grad_zero_"),
     (torch._C._get_tracing_state, "aten::_get_tracing_state"),
     (warnings.warn, "aten::warn"),
+    (torch._C._VariableFunctions.stft, "aten::stft")
```

Comment:
if you do the above you shouldn't need this here, instances in `nn/functional.py` resolve to their `aten::` equivalent already 

https://github.com/pytorch/pytorch/blob/cfb4862673303a2725d815c0338435f6b6c699fb/torch/nn/functional.py#L805-L806

Reply:
For some reason I had to add _VF.stft here, now sure why. But I moved _VF so that i can just use `_VF.stft` inline.